index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio-Setup</title>
+</head>
+<body>
+
+    <!-- <header>
+        <nav class="header-logo">
+            <h2>My Logo</h2>
+            <img src="images/Enabled.png" alt="" class="image-icon">
+        </nav>
+    </header> -->
+
+    <header class="header-logo">
+        <h2>My Logo</h2>
```

Comment:
- [ Optional ] Kindly i recommend you to use `a` tag for the logo, This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. As you implemented as first before  you make the change , when you moved the log outside the `nav` tag please use` a` tag for logo for semantic purpose Just like this : 
```
<header>
        <a class="mylogo" href="#home">My Logo</a>
        <nav>
           <img src="images/Enabled.png" alt="" class="image-icon">
        </nav>
    </header>
```